---
title: "qPCR for Nickel exposure on *Botryllus schlosseri*"
subtitle: "Relative standard curves to determine primer effenciencies."
author: "Celeste Valdivia"
date: "2024-02-12"
categories: [nickel chloride, qPCR, primers]
image: "EP PCR Gel Results 12_18_23 -2.jpg"
---

# Objective

Analysis of gene expression with quantitative Polymerase Chain Reaction (qPCR) will be a component of the *in vivo* nickel exposure work conducted in autumn 2023. Four experiments were completed using a high and low dose of nickel chloride (5 and 45 mg L^-^) and we are aiming to evaluate the impact of nickel toxicity on gene expression relating to oxidative stress, senescence, stemness, and apoptosis.

# Background

Refer to [pilot exposure post](https://valeste.github.io/tough-tunicates/posts/nicl2-botryllus-pilot/) for more details regarding procedures and experimental design.

## RNA Extract Information

After gDNA elimination with enzymatic kit.

|    Date    	|       Sample ID      	| RNA Concentration (ng/uL) 	| A260/A280 	| 260(10mm) 	|
|:----------:	|:--------------------:	|:-------------------------:	|:---------:	|:---------:	|
| 09/30/2023 	| 9302023 0-R1 pilot   	|                      19.2 	|      1.55 	|      0.48 	|
| 09/30/2023 	| 9302023 0-R2 pilot   	|                     113.2 	|      1.85 	|      2.83 	|
| 09/30/2023 	| 9302023 0-R3 pilot   	|                     152.4 	|      1.87 	|     3.809 	|
| 09/30/2023 	| 9302023 0-R4 pilot   	|                      67.1 	|      1.77 	|     1.676 	|
| 09/30/2023 	| 9302023 5-R1 pilot   	|                      22.4 	|      1.66 	|      0.56 	|
| 09/30/2023 	| 9302023 5-R2 pilot   	|                      30.6 	|      1.68 	|     0.764 	|
| 09/30/2023 	| 9302023 5-R3 pilot   	|                        29 	|      1.67 	|     0.726 	|
| 09/30/2023 	| 9302023 5-R4 pilot   	|                      45.1 	|      1.77 	|     0.128 	|
| 09/30/2023 	| 9302023 45-R2 pilot  	|                      87.6 	|      1.82 	|      2.14 	|
| 09/30/2023 	| 9302023 45-R3 pilot  	|                      13.8 	|      1.47 	|           	|
| 09/30/2023 	| 9302023 45-R4 pilot  	|                      84.8 	|      1.83 	|      2.12 	|
| 10/25/2023 	| 10252023 0-R1 E1     	|                      12.1 	|      1.55 	|     0.305 	|
| 10/25/2023 	| 10252023 0-R2 E1     	|                      45.9 	|      1.83 	|     1.148 	|
| 10/25/2023 	| 10252023 0-R3 E1     	|                      78.4 	|      1.85 	|     1.959 	|
| 10/25/2023 	| 10252023 0-R4 E1     	|                      38.5 	|       1.8 	|     0.962 	|
| 10/25/2023 	| 10252023 5-R1 E1     	|                      83.3 	|      1.82 	|     2.082 	|
| 10/25/2023 	| 10252023 5-R3 E1     	|                      27.1 	|      1.69 	|     0.676 	|
| 10/25/2023 	| 10252023 5-R4 E1     	|                      69.5 	|      1.84 	|     1.737 	|
| 10/25/2023 	| 10252023 45-R1 E1    	|                        50 	|      1.78 	|     1.249 	|
| 10/25/2023 	| 10252023 45-R2 E1    	|                      57.3 	|      1.84 	|     1.433 	|
| 10/25/2023 	| 10252023 45-R3 E1    	|                      43.5 	|      1.73 	|     1.087 	|
| 10/25/2023 	| 10252023 45-R4 E1    	|                      60.1 	|      1.83 	|     1.505 	|
| 11/15/2023 	| 11152023 0-R1 E2     	|                      86.5 	|      1.86 	|     2.163 	|
| 11/15/2023 	| 11152023 0-R2 E2     	|                      54.7 	|      1.66 	|     1.367 	|
| 11/15/2023 	| 11152023 0-R3 E2     	|                      37.4 	|      1.72 	|     0.935 	|
| 11/15/2023 	| 11152023 5-R1 E2     	|                      58.4 	|      1.66 	|     1.459 	|
| 11/15/2023 	| 11152023 5-R2 E2     	|                      69.7 	|      1.69 	|     1.743 	|
| 11/15/2023 	| 11152023 5-R3 E2     	|                      33.7 	|      1.51 	|     0.843 	|
| 11/15/2023 	| 11152023 45-R2 E2    	|                      34.5 	|      1.44 	|     0.863 	|
| 11/15/2023 	| 11152023 45-R3 E2    	|                      16.2 	|       1.5 	|     0.405 	|
| 11/29/2023 	| 11292023 0-R1 E3     	|                      72.4 	|      1.78 	|     1.809 	|
| 11/29/2023 	| 11292023 0-R2 E3     	|                      60.3 	|      1.78 	|     1.507 	|
| 11/29/2023 	| 11292023 0-R3 E3     	|                      21.4 	|      1.61 	|     0.534 	|
| 11/29/2023 	| 11292023 5-R1 E3     	|                      12.7 	|      1.43 	|     0.318 	|
| 11/29/2023 	| 11292023 5-R2 E3     	|                      15.5 	|      1.57 	|     0.387 	|
| 11/29/2023 	| 11292023 5-R3 E3     	|                      64.4 	|      1.76 	|     1.609 	|
| 11/29/2023 	| 11292023 45-R1 E3    	|                      80.6 	|      1.82 	|     2.016 	|
| 11/29/2023 	| 11292023 45-R2 E3    	|                      49.3 	|      1.77 	|     1.231 	|
| 11/29/2023 	| 11292023 45-R3 E3    	|                      64.3 	|      1.79 	|     1.607 	|                   	|


## cDNA Pool

| Date Made  	| Sample        	| cDNA ng/uL 	| A260/A280 	|
|------------	|---------------	|------------	|-----------	|
| 12/18/2023 	| Original Pool 	|     2231.2 	|      1.73 	|
|  1/14/2024 	|          10^-2^ 	|     223.12 	|      1.73 	|
|  1/14/2024 	|          10^-3^ 	|     22.312 	|      1.73 	|
|  1/14/2024 	|          10^-4^ 	|      2.312 	|      1.73 	|
|  1/14/2024 	|          10^-5^ 	|     0.2312 	|      1.73 	|
|  1/14/2024 	|          10^-6^ 	|    0.02312 	|      1.73 	|


## Primers
| Number | Gene     | Forward Primer (5'-3')      | Reverse Primer (5'-3')     | Source                          |
|--------|----------|-----------------------------|----------------------------|---------------------------------|
|      1 | BAX      | CATTGACCTTGCGTGTTATTGG      | AAGATGTCCTTGGTGGGTTG       | in house                        |
|      2 | POU3     | CAGTAGGTCTTCGCATCCATC       | CGTTTGCATCGTTGGTGTAAG      | in house                        |
|      3 | PolB     | GGCTTCACAATCAATGAGTATTCC    | AGCAATCTTTCCGCATTCATTC     | in house                        |
|      4 | TNF      | GTTTCAGTCAACACCTTGTGTC      | AATCTCGACCGCTGACTTATAC     | in house                        |
|      5 | SOXB1    | GGGAATGTGTTCATCCTCTGG       | TGAATCCAGGAGTTACTCAAGTTC   | in house                        |
|      6 | MYC      | GAAGTCCTGGCTAAGATGCTAC      | AGCGTCAAGAAGGAAGTACG       | in house                        |
|      7 | AIF1     | GGATTCTCTCACCGATGGAATAG     | CCTGACTTGGAGCCTTTATGG      | in house                        |
|      8 | SOD1     | CAATCCATATAACAAGGTTCATGGC   | TGATATCCACACAGGCACAAC      | in house                        |
|      9 | PARP1    | AATTTGGCACCGCTCATTTC        | TGCAAGACCTTCAGCTTCTC       | in house                        |
|     10 | SIRT6    | ATTAGTACCGCGTCTGGAATAC      | GGATGGAATCGCTTCTTCAAATC    | in house                        |
|     11 | IAP7     | GGTCTCTTCACGGACATCTTC       | GCTCAGTATGTTGTGCTTGTTC     | in house                        |
|     12 | HSP70    | CGAGAAGTTGAAGGACAAGATCTCCG  | TCTCTGCAGTCTGGTTGTTGTCGAGC | doi:10.1016/j.ydbio.2017.10.023 |
|     13 | Mortalin | TTCATGATACCGAAACCAAGATGGACG | AATTTCCTCCTTGATGGCATCCACC  | doi:10.1016/j.ydbio.2017.10.023 |
|     14 | ACTb     | GTAGGTAGTCTCGTGAATTC        | CACGCCATCTTGCGTCTGGA       | doi:10.1016/j.dci.2006.12.009   |
|     15 | 18s      | GGCAGCCTCCGGGAAACCAAAGTC    | CTGGTGGTGCCCTTCCGTCAATTC   | doi:10.1016/j.ydbio.2017.10.023 |

# End-Point PCR

::: {#fig-ep-pcr layout-ncol="2"}
![](EP PCR Gel Results 12_18_23 -1.jpg)

![](EP PCR Gel Results 12_18_23 -2.jpg)

EP-PCR conducted to confirm correct size of amplicon products. In this PCR reaction run, primer sets were at a concentration of 5 uM and the reagent used was SYBR PowerTrack. Note this is different than what was used for the 1:10 dilution series relative standard curves. 
:::


# qPCR Reaction Information

| **Component**                                        | **per 1 rxn (uL)** |
|------------------------------------------------------|--------------------|
| Applied Biosystems™ Power SYBR™ Green PCR Master Mix | 5                  |
| Nuclease-free water                                  | 3.5                |
| Forward Primer                                       | 0.25               |
| Reverse Primer                                       | 0.25               |
| cDNA                                                 | 1                  |
| **Total**                                            | **10**             |

# Relative Standard Curves

## Good

### Beta actin
  
::: {#fig-ep-pcr layout-ncol="3"}

![](png/actb_mc.png)

![](png/actb_sc.png)

![](png/actb_amp.png)

Efficiency for beta actin is 94%

:::

### 18s


::: {#fig-ep-pcr layout-ncol="3"}

![](png/18s_mc.png)

![](png/18s_sc.png)

![](png/18s_amp.png)

Efficiency value for 18s = 89%.
:::

## Questionable

### BAX
  
::: {#fig-ep-pcr layout-ncol="3"}

![](png/bax_mc.png)

![](png/bax_sc.png)

![](png/bax_amp.png)

Efficiency for BAX is 350%

:::

### AIF1
  
::: {#fig-ep-pcr layout-ncol="3"}

![](png/aif1_mc.png)

![](png/aif1_sc.png)

![](png/aif1_amp.png)

Efficiency for AIF1 is TBD.

:::

### Mortalin

::: {#fig-ep-pcr layout-ncol="3"}

![](png/mort_mc.png)

![](png/mort_sc.png)

![](png/mort_amp.png)

Efficiency for Mortalin is TBD.

:::

### SIRT6

::: {#fig-ep-pcr layout-ncol="3"}

![](png/sirt6_mc.png)

![](png/sirt6_sc.png)

![](png/sirt6_amp.png)

Efficiency for SIRT6 is more than 2 million.

:::

### SOD1

::: {#fig-ep-pcr layout-ncol="3"}

![](png/sod1_mc.png)

![](png/sod1_sc.png)

![](png/sod1_amp.png)

Efficiency for SOD1 is TBD.

:::

### POU3

::: {#fig-ep-pcr layout-ncol="3"}

![](png/pou3_mc.png)

![](png/pou3_sc.png)

![](png/pou3_amp.png)

Efficiency for POU3 is 673%.

:::

### SOXB1

::: {#fig-ep-pcr layout-ncol="3"}

![](png/soxb1_mc.png)

![](png/soxb1_sc.png)

![](png/soxb1_amp.png)

Efficiency for soxb1 is TBD.

:::

### MYC

::: {#fig-ep-pcr layout-ncol="3"}

![](png/myc_mc.png)

![](png/myc_sc.png)

![](png/myc_amp.png)

Efficiency for MYC is TBD.

:::

### IAP7

::: {#fig-ep-pcr layout-ncol="3"}

![](png/iap7_mc.png)

![](png/iap7_sc.png)

![](png/iap7_amp.png)

Efficiency for IAP7 is TBD.

:::

## Trash

### PolB

::: {#fig-ep-pcr layout-ncol="3"}

![](png/polb_mc.png)

![](png/polb_sc.png)

![](png/polb_amp.png)

Basically no amplification.

:::

### TNF

::: {#fig-ep-pcr layout-ncol="3"}

![](png/tnf_mc.png)

![](png/tnf_sc.png)

![](png/tnf_amp.png)

Basically no amplification.

:::

### HSP70

::: {#fig-ep-pcr layout-ncol="3"}

![](png/hsp70_mc.png)

![](png/hsp70_sc.png)

![](png/hsp70_amp.png)

Bascially no amp.

:::

### PARP1

::: {#fig-ep-pcr layout-ncol="3"}

![](png/parp1_mc.png)

![](png/parp1_sc.png)

![](png/parp1_amp.png)

Late amp, maybe rare in my samples? 


:::


# Discussion

## cDNA has low purity?

According to online sources DNA purity (260/280) should fall above 1.8. Currently the cDNA is at 1.7. RNA should fal above 2.0 (260/280). The RNA after gDNA elimination fell below that at an **average of 1.71 (SD ± 0.12)**.

When running all the primer sets at a 1:5 dilution series, higher concentrations of cDNA resulted in a 'horseshoe' shape (indicative of failure to fluoresce) and most diluted sample (1:625) had lower Ct values than higher concentrations (1:125). Upon running a 1:10 dilution series, this issue is no longer as apparent. However, online sources mention that an amplification efficiency above 100% (which is what we see for nearly all the primers) or a really flat slope on the standard curve is indicative of polymerase inhibition.


::: {#fig-1to5 layout-ncol='1'}

![](png/amp_plot_plate3.png)

18s primer amplification plot at a 1:5 dilution series. Downward parabola shapes are higher concentrations of the cDNA pools.

:::


**Common contaminants**:

-   hemoglobin

-   polysaccharides

-   proteinase K

-   ethanol (during the RNA isolation)

However, the purity of my RNA prior to gDNA elimination was on **average 1.89 (SD ± 0.11)** so likely reagents fromt the gDNA elimination step were carried over into my samples.

## Primers are not optimally designed?

Only workable primer sets will be those with a singular peak and appear to fail to have an acceptable efficiency curve due to high standard deviation across replicates.


The following primers currently in stock will be utilized moving forward:

### Workable Primers

| Number | Gene     | Forward Primer (5'-3')      | Reverse Primer (5'-3')     | Source                          |
|--------|----------|-----------------------------|----------------------------|---------------------------------|
|      1 | **BAX**      | CATTGACCTTGCGTGTTATTGG      | AAGATGTCCTTGGTGGGTTG       | in house                        |
|      7 | **AIF1** | GGATTCTCTCACCGATGGAATAG     | CCTGACTTGGAGCCTTTATGG      | in house                        |        
|     10 | **SIRT6**    | ATTAGTACCGCGTCTGGAATAC      | GGATGGAATCGCTTCTTCAAATC    | in house                        |
|     13 | **Mortalin** | TTCATGATACCGAAACCAAGATGGACG | AATTTCCTCCTTGATGGCATCCACC  | doi:10.1016/j.ydbio.2017.10.023 |
|     14 | **ACTb**     | GTAGGTAGTCTCGTGAATTC        | CACGCCATCTTGCGTCTGGA       | doi:10.1016/j.dci.2006.12.009   |
|     15 | **18s**      | GGCAGCCTCCGGGAAACCAAAGTC    | CTGGTGGTGCCCTTCCGTCAATTC   | doi:10.1016/j.ydbio.2017.10.023 |


Further literature search regarding previous work conducted with *B. schlosseri* and qPCR resulted in finding the following paper:

::: callout-note

#### Citation

Rosner A, Kravchenko O, Rinkevich B. IAP genes partake weighty roles in the astogeny and whole body regeneration in the colonial urochordate Botryllus schlosseri. Developmental Biology. 2019 Apr;448(2):320–41. [doi.org/10.1016/j.ydbio.2018.10.015](https://doi.org/10.1016/j.ydbio.2018.10.015)

:::

This paper utilizes several primer sets targeting genes in relation to apoptosis. The following primer sets are pulled from this paper:

### Proposed Primers

| Gene* or Accession** number 	| Gene name 	| Forward primer 	| Reverse primer 	| Source 	|
|:---:	|:---:	|:---:	|:---:	|---	|
| g5785 	| IAP6 	| CTGGTATTGTAACGGCGGCCTGCAA 	| GGTGAACGAAATGCTGGCCTTTGGA 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g7904 	| IAP8 	| ATTTACCTCGGACGTTAGCGCAGGAC 	| CCCGGAACAAGAGAAGCATTGAACTC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g7149 	| IAP9 	| GGATGCGGGATATGCGGAGGATAG 	| AGCTTCCGCCGACTGATAGCATG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g27485 	| IAP14 	| GCGTGACCGTGTGAAATGCTGG 	| CCACTTTGCATGCTCGACCCAC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g44848 	| **IAP15** 	| ATGGAAGCCACCGACGAGATGAG 	| TCGATAACGTTCTGTTGGCAGCTCTTG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g56072 	| IAP18 	| TAAGCACATTCATGCGGTGGTCG 	| GCCCAAATAGAACAATCCGCAGTCTG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g29315 	| IAP21 	| CGTGAGCCTCTACTGCGCACTTCTC 	| GCTTGAGTAGAAGTTGGCGCAGAGC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| KU948200.1** 	| Bax 	| ATCTAGGCCAACCCGACCGAGAACC 	| CAACAGCTCCCTGGACCAAAGACG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| KU948202.1** 	| AIF1 	| GGTGAGCAAGGGCATAGAAC 	| TGAACGACTCCGTGTTTGG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g8767 	| caspase 2 	| TGGTGAAGGTCCTGGCGAAGAAC 	| GCCTTTAACGACGCTGTTGACGG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g11647 	| caspase7.1 	| GCACAAATTCCGTGGTTGCTTCC 	| TTCTCTTGTCTCTGTGCGTCCCGTTTC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g3334 	| **caspase9** 	| GAAACCGATCTTACCGCCTACAACATG 	| ACGCATATATCAGATCGTCCTTGTTGC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g49756 	| AKT1 	| CTTGAAGACAATGATTACGGCAGGTCC 	| AAGCGTCCGCACATCATCTCGTAC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g1445 	| PIK3CA 	| CTGATAAGGAAAGTCCACGGCAACGAC 	| CTGACCGGTTTGACGTGGAGTAATTG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g42386 	| GSK3b2 	| GTTTCCGCAAATCAAGGCTCATCC 	| TGCTACATAAACCGATGGCGTCCTG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g12744 	| cdc25 	| GAAATGTTCCCTGCCCACTGTGAC 	| CGATGCCGTGTTCTTTCTTCAGAGC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g41847 	| myc 	| GCAGATCAGGGCCGCTCACAATGTC 	| TGTCGCGCAGCGTCAAGAAGGAAG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g61081 	| sox2 	| ACGCAGTTAATCATCATCCATCTCACC 	| AACCGGGATGTGGGCTACTTGC 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g68340 	| hsp90α 3_1 	| AAGATCATTTAGCCGTCAAGCACTTCTC 	| GGGCACGCTTTGGAACGAAGAG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|
| g67511 	| hsp90β 2_4 	| AAGGCCCAGGCTCTTCGTGACTCTTC 	| GAGTGTTCGGGATTGATCTCCAAGTG 	| doi.org/10.1016/j.ydbio.2018.10.015 	|

::: callout-note

*Gene number assigned by the Botrullus Genome Project (Voskoboynik et al., 2013).

**Accession number assigned by the NCBI GenBank.

:::

Of the suite of IAPs reduced from the ones used in the referenced paper to capture those that were most interesting under oxidative stress at stage A and across blastogenic stages. 

In our study we did not target a specific stage for oxidative stress (nickel) exposures so I believe it would be interesting to capture IAPs impacted by both factors.


