---
title: "Troubleshoot qPCR for Nickel exposure on *Botryllus schlosseri*"
subtitle: "re: RobertsLab/resources issue #1834 "
author: "Celeste Valdivia"
date: "2024-03-06"
categories: [nickel chloride, qPCR, primers, issues]
---

# Background

In response to @kubu4's questions in issue [#1834]((https://github.com/RobertsLab/resources/issues/1834)) opened in Roberts Lab resources repo.

# Response

**RNA**

> 1. Do you have access to a Qubit (the Roberts Lab has one, if you don't already)? The Qubit is far more sensitive and accurate than the NanoDrop, but will not provide you with A260/280 and A260/230 ratios.

No, I don't currently have access to a Quibit out at UW Tacoma.

> 2.  Do you have the A260/230 ratios? This information will provide a clearer picture on whether or not there are any possible inhibitors that got carried over (things like phenol, alcohol, excessive salts result in a poor A260/230 ratio).

The NanoDrop available for use at UW Tacoma is a NanoDrop Lite and only reports 260/280 ratios. I agree it would be good to have info for the 260/230 ratios as well.

**DNase**

> 1a. Which DNase protocol did you follow - standard or rigorous?

Standard

> 1b. What was the makeup of your DNase reactions (e.g. 1uL DNase, 5ug of RNA, 25uL water, etc)?

| **gDNA Elimination Component** | **per 1 rxn (uL)** |
|--------------------------------|--------------------|
| 10x DNase Buffer               | 2                  |
| Turbo Enzyme                   | 1                  |
| RNA sample                     | 20                 |
| **Total**                      | **23**             |

**cDNA**

> 1.  How much RNA did you use for your reverse transcription reactions?

96.8 ng per sample rxn

> 2.  What was the makeup of your reverse transcription reactions (e.g. 5uL buffer, 5ug RNA, 10uL, 1uL dNTPs)?

| **cDNA synthesis Component**   | **per 1 rxn (uL)** |
|--------------------------------|--------------------|
| SuperScript IV VILO Master Mix | 2                  |
| Nuclease-free water            | variable           |
| RNA sample                     | variable           |
| **Total**                      | **8**              |


**qPCR**

>1.What are your expected amplicon sizes for each primer set? I did see a note in your notebook entry on primer design that said you were targeting 500bp, but it'd be helpful/useful to include that info in your table(s).

Below is the info for amplicon length per primer set. I couldn't find all the info for primers pulled from the literature but aside from ACTb all are below 200 bp as I described for my aims in designing.


| Number 	| Gene     	| Forward Primer (5'-3')      	| Reverse Primer (5'-3')     	| NCBI Accession Number (If complete CDS) 	| Botryllus Contig Sequence (if no complete CDS was found) 	| Amplicon Length 	| Source                          	|
|--------	|----------	|-----------------------------	|----------------------------	|-----------------------------------------	|----------------------------------------------------------	|-----------------	|---------------------------------	|
| 1      	| BAX      	| CATTGACCTTGCGTGTTATTGG      	| AAGATGTCCTTGGTGGGTTG       	| KU948200.1                              	| None                                                     	| 97              	| in house                        	|
| 2      	| POU3     	| CAGTAGGTCTTCGCATCCATC       	| CGTTTGCATCGTTGGTGTAAG      	| KU948203.1                              	| None                                                     	| 80              	| in house                        	|
| 3      	| PolB     	| GGCTTCACAATCAATGAGTATTCC    	| AGCAATCTTTCCGCATTCATTC     	| None                                    	| botctg092056                                             	| NA              	| in house                        	|
| 4      	| TNF      	| GTTTCAGTCAACACCTTGTGTC      	| AATCTCGACCGCTGACTTATAC     	| None                                    	| botctg065850                                             	| 131             	| in house                        	|
| 5      	| SOXB1    	| GGGAATGTGTTCATCCTCTGG       	| TGAATCCAGGAGTTACTCAAGTTC   	| KU948203.1                              	| None                                                     	| 116             	| in house                        	|
| 6      	| MYC      	| GAAGTCCTGGCTAAGATGCTAC      	| AGCGTCAAGAAGGAAGTACG       	| KU948203.1                              	| None                                                     	| 119             	| in house                        	|
| 7      	| AIF1     	| GGATTCTCTCACCGATGGAATAG     	| CCTGACTTGGAGCCTTTATGG      	| KU948202.1                              	| None                                                     	| 102             	| in house                        	|
| 8      	| SOD1     	| CAATCCATATAACAAGGTTCATGGC   	| TGATATCCACACAGGCACAAC      	| EE743567.1                              	| None                                                     	| 107             	| in house                        	|
| 9      	| PARP1    	| AATTTGGCACCGCTCATTTC        	| TGCAAGACCTTCAGCTTCTC       	| KU948201.1                              	| None                                                     	| 94              	| in house                        	|
| 10     	| SIRT6    	| ATTAGTACCGCGTCTGGAATAC      	| GGATGGAATCGCTTCTTCAAATC    	| None                                    	| botctg109566                                             	| 114             	| in house                        	|
| 11     	| IAP7     	| GGTCTCTTCACGGACATCTTC       	| GCTCAGTATGTTGTGCTTGTTC     	| KU948203.1                              	| None                                                     	| 111             	| in house                        	|
| 12     	| HSP70    	| CGAGAAGTTGAAGGACAAGATCTCCG  	| TCTCTGCAGTCTGGTTGTTGTCGAGC 	| NA                                      	| NA                                                       	| NA              	| doi:10.1016/j.ydbio.2017.10.023 	|
| 13     	| Mortalin 	| TTCATGATACCGAAACCAAGATGGACG 	| AATTTCCTCCTTGATGGCATCCACC  	| KY212120.1                              	| None                                                     	| 79              	| doi:10.1016/j.ydbio.2017.10.023 	|
| 14     	| ACTb     	| GTAGGTAGTCTCGTGAATTC        	| CACGCCATCTTGCGTCTGGA       	| NA                                      	| NA                                                       	| 321             	| doi:10.1016/j.dci.2006.12.009   	|
| 15     	| 18s      	| GGCAGCCTCCGGGAAACCAAAGTC    	| CTGGTGGTGCCCTTCCGTCAATTC   	| NA                                      	| NA                                                       	| NA              	| doi:10.1016/j.ydbio.2017.10.023 	|

> What was the makeup of your end-point PCR reactions (e.g. 10uL 2x polymerase, 0.5uL 10uM Pf, 0.5uL Pr, 9uL H2O, etc)?

| **EP-PCR Component**                                 | **per 1 rxn (uL)** |
|------------------------------------------------------|--------------------|
| PCR Master Mix (brand TBD)                           | 12.5               |
| Nuclease-free water                                  | 9.5                |
| Forward Primer (5 uM)                                | 1                  |
| Reverse Primer (5 uM)                                | 1                  |
| 1:3 dilution of pooled (per treatement) cDNA         | 1                  |
| **Total**                                            | **10**             |


> What were your cycling parameters for end-point PCR?

Need to double check but roughly these parameters.


| Stage      	| Step          	| Temperature (C) 	| Duration (sec) 	| Cycles 	|
|------------	|---------------	|-----------------	|----------------	|--------	|
| PCR        	| Denature      	| 95              	| 15             	| 40     	|
| PCR        	| Anneal/Extend 	| 60              	| 60             	| 40     	|
| Hold        | Hold            | 4                 | infinity        | 1       |

> Where is your 18s primer set on either of the two gels?

Gel annotation is mislabeled. Where it says 40s it should say 18s.

> What were your cycling parameters for your qPCR reactions?

| Stage      	| Step          	| Temperature (C) 	| Duration (sec) 	| Cycles 	|
|------------	|---------------	|-----------------	|----------------	|--------	|
| Hold       	| Hold          	| 50              	| 120            	| 1      	|
| Hold       	| Hold          	| 95              	| 600            	| 1      	|
| PCR        	| Denature      	| 95              	| 15             	| 40     	|
| PCR        	| Anneal/Extend 	| 60              	| 60             	| 40     	|
| Melt Curve 	| Denature      	| 95              	| 15             	| 1      	|
| Melt Curve 	| Anneal/Extend 	| 60              	| 60             	| 1      	|
| Melt Curve 	| Dissociation  	| 95              	| 1              	| 1      	|



> Were your NTCs clean? It be cool if you could color code your amplification plots to have the NTCs be a different color than the rest of the samples - that would make it easier to quickly evaluate this aspect of things.

The only NTCs that were "clean" were the ones for the primers I labeled as trash that failed to amplify in target reactions with cDNA. Beta actin also had a clean NTC.

Note that for some of the GOI's like AIF1 and SOD1, the NTC Ct value is nearly identical to the rest of the Ct values. (Meaning the Ct value remains the same across decreasing concentrations of cDNA and in the NTC, likely a primer dimer artifcat and no amplification potentially occuring?)

| Target Name 	| Task 	| CT           	|
|-------------	|------	|--------------	|
| BActin      	| NTC  	| Undetermined 	|
| BAX         	| NTC  	|       32.548 	|
| SOD1        	| NTC  	|       26.921 	|
| SIRT6       	| NTC  	| Undetermined 	|
| IAP7        	| NTC  	|       32.048 	|
| HSP70       	| NTC  	| Undetermined 	|
| Mortalin    	| NTC  	|       32.660 	|
| 18s         	| NTC  	|       32.623 	|
| POU3        	| NTC  	|       33.793 	|
| PolB        	| NTC  	| Undetermined 	|
| TNF         	| NTC  	| Undetermined 	|
| MYC         	| NTC  	|       32.153 	|
| AIF1        	| NTC  	|       27.873 	|
| PARP1       	| NTC  	| Undetermined 	|

